_projects/enhance-and-improve-coala-community-and-projects-website.md

```diff
@@ -0,0 +1,160 @@
+maintainers, this project includes the development of form with some
+rudimentary checks that will promote the coala-newcomer to coala-developer.
+Moreover, this project adds major enhancements to websites that will help
+newcomers as well as developers to know more about this community and about
+what they have contributed to it.
+
+### Goals
+
+* Add calendar app to show timeline for various events
+* Add a one-stop tab for all google forms
+* Add contributors geolocation
+* Add *Login/SignUp* using GitHub
+* Add organization teams
+* Advanced filters
+* Automate welcoming process for newcomer
+* Automate newcomer promotion
+* Create new database models to manage data
+* Ease navigatibility in websites
+* Improve the **UI/UX of websites** to make more interactive
+* Motivating newcomers & developers with -
+  * modifications in Gamification Leaderboard
+  * adding a tab of **GSoC** in projects website
+* New coala-community logo
+* Contributors profile page
+* Admin/Maintainers dashboard
+
+### Milestones
+
+#### PREPARATION/BONDING
+
+* Detailed research on what **new cron jobs** or **API endpoints** need to be
+  added in [coala webservices](https://gitlab.com/coala/landing/).
+* Discussion on improvements to be made in coala [community website](http://community.coala.io/) and
+  [projects website](http://projects.coala.io/)
+* Research on new models to be created (if any) and adding new fields to
+  existing db models in coala community website **db models**.
+* *Created* and *Merged* a cEP of how the new enhancements will be implemented
+  and how will it be beneficial to [coala community](https://github.com/orgs/coala/teams/coala-newcomers).
+
+#### CODING PHASE 1
+
+1. Implementation of methods in **coala [webservices](https://gitlab.com/coala/landing)**
+2. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create new **[Community Logo](https://github.com/coala/community/issues/55) + Header Image of README.md**
+
+    b. Add information about new endpoints at coala-webserivces homepage.
+
+    c. Add [contributor's Geolocation](https://github.com/coala/community/issues/18)
+
+    d. Add [Organisation Teams](https://github.com/coala/community/issues/12)
+
+    e. Add [Mentors](https://github.com/coala/community/issues/11)
+
+    f. Add **Newcomer Invitation** [form](https://github.com/coala/community/issues/89#issuecomment-468955279)
+
+    g. Add **Promotion to Developer** [form](https://github.com/coala/community/issues/89)
+
+    h. Manage live data of community members
+
+    i. Addition of more data fields in **Users** db model
+
+    j. Add **Login and Sign-up** using GitHub
+3. Make Improvements (or fix bugs) in coala [community website](https://community.coala.io/)
+
+    a. Build failure for more than one remote [#235](https://github.com/coala/community/issues/235)
+
+    b. Netlify build failure for fork [#232](https://github.com/coala/community/issues/232)
```

Comment:
These are just minor fixes, not sure if they qualify to be included in a gsoc project. This could be a pre gsoc or community bonding task.